### ISSUE TITLE
screenreader accessibility

### ISSUE THREAD

**Issue by Veyfeyken** [FrDH/mmenu-js#1201]
Hi, 

Regarding this demo:
https://mmenujs.com/demo/index.html?demo=menu-advanced-1

It's not accessible with a screenreader unfortunately. 

The hamburger button has no accessible name, it's an empty link. Suggestion: use a button and add hidden text 'menu' inside it.

Activating the button opens the menu but the menu is not immediately below the button in the reading order (= order in HTML). The menu is above the hamburger button in the HTML. A screenreader user will not encounter it when continuing to navigate. They would have to navigate backwards to find it, which is very unlikely. 

2 possible solutions:
* Change the order of the HTML, place the content of the menu directly below the button that opens it (preferred and most robust solution)
* Implement a focus jump when the hamburger is initiated, use JavaScript focus() to set focus to the first element inside the menu.

Kind regards,

Gijs

**Comment by FrDH** [comment]
The demo files are not optimized for accessibility, I'll see if I can do so later on.
